docs/index.bs

```diff
@@ -1862,7 +1862,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
         1. Let |requests| be an array containing only |request|.
         1. Let |responseArrayPromise| be the result of running the algorithm specified in {{Cache/addAll(requests)}} passing |requests| as the argument.
-        1. Return the result of [=upon settling|transforming=] |responseArrayPromise| with a fulfillment handler that returns undefined.
+        1. Return the result of [=promise/reacting=] to |responseArrayPromise| with a fulfillment handler that returns undefined.
```

Comment:
The GitHub preview doesn't show this hyperlinked, is it just because it's hasn't caught up to the latest WebIDL change?

Reply:
Yeah, I'll give it a day or so before merging.